lib/web_ui/lib/src/engine/compositor/util.dart

```diff
@@ -187,12 +187,16 @@ js.JsArray<double> makeSkMatrix(Float64List matrix4) {
   return skMatrix;
 }
 
+const double kLightHeight = 600.0;
+const double kLightRadius = 800.0;
```

Comment:
Let's document where these constants come from, in case we need to update them in the future.

Reply:
Done.